Framework/Beamline/test/ComponentInfoTest.h

```diff
@@ -910,32 +909,11 @@ class ComponentInfoTest : public CxxTest::TestSuite {
     auto &b = *std::get<0>(infos2);
     a.setScanInterval({0, 1});
     b.setScanInterval({0, 1});
-    b.setScanInterval({0, 1});
     TS_ASSERT_THROWS_EQUALS(a.merge(b), const std::runtime_error &e,
                             std::string(e.what()),
                             "Cannot merge ComponentInfo: size mismatch");
   }
 
-  void test_merge_fail_no_intervals() {
```

Comment:
Why was this test removed? I vaguely remember that we discussed this, but what was the reasoning?

Reply:
Because the scan intervals are now always defined, with a default single interval `{0,1}`